app/mailers/user_mailer.rb

```diff
@@ -1,16 +1,21 @@
 class UserMailer < ActionMailer::Base
-  default from: "from@example.com"
+  default from: 'us@supermarkit.ca'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.